exotic/exotic.py

```diff
@@ -1933,13 +1946,23 @@ def main():
         if not args.realtime:
             real_time_dir = user_input("Enter the Directory Path of imaging files: ", type_=str)
         else:
-            real_time_dir = args.realtime
+            real_time = Path(args.realtime)
+            with real_time.open('r') as json_file:
+                info = json.load(json_file)
+                real_time_dir = info['user_info']['Directory with FITS files']
+                targetName = info['planetary_parameters']['Planet Name']
+                exotic_UIprevTPX = info['user_info']['Target Star X & Y Pixel'][0]
+                exotic_UIprevTPY = info['user_info']['Target Star X & Y Pixel'][1]
+                exotic_UIprevRPX = info['user_info']['Comparison Star(s) X & Y Pixel'][0][0]
+                exotic_UIprevRPY = info['user_info']['Comparison Star(s) X & Y Pixel'][0][1]
+
```

Comment:
EXOTIC will grab the first comparison star coordinates used in the inits.json file.